docs/medical-api/more-info/webhooks.mdx

```diff
@@ -96,6 +100,12 @@ Example payload:
       ...
     }
   ]
+  "documentRequestMetadata": {
+    "<some-request-meta-property>": "<the-property-value>"
+  },
+  "consolidatedRequestMetadata": {
+    "<some-request-meta-property>": "<the-property-value>"
+  },
```

Comment:
why not just put this in an optional `data` field in the webhook `meta` base model?

this sets a pattern for all WHs in our API, so not sure why we'd want a bunch of permutations like `documentRequestMetadata`, for each type - this seems to make it more cumbersome for devs

lmk what you think